src/test/emulators/functionsEmulatorRuntime.spec.ts

```diff
@@ -246,6 +244,7 @@ describe("FunctionsEmulator-Runtime", () => {
           return {
             function_id: require("firebase-functions").https.onRequest((req: any, res: any) => {
               res.json(admin.firestore()._settings);
+              return Promise.resolve();
```

Comment:
Was this required?  Normally `res.json` is enough.

Reply:
Ah! This is required to make the tests pass.

If a value or a promise isn't returned in a function, the SDK logs a line saying "Function didn't return a promise or a value. Please do that.". This breaks the assertion below that's reading the log lines because that string isn't JSON parsable.

(This was the bug that I was tracking down while `nyc` was hiding some stuff too. fun times)

Reply:
Gotcha.

Reply:
The answer is really consistency. They all _should_ return a promise or a value. Some of the ones I didn't change don't check logs, so it doesn't matter as much.